ks-frontend-services/src/auth/FirebaseAuthService.ts

```diff
@@ -98,6 +98,12 @@ class FirebaseAuthService {
     return await user.getIdToken(true);
   };
 
+  refreshToken = async (token: string, forceRefresh = true) => {
+    // If forceRefresh is true, exchange token + resolve the token
+    // Refreshes the token
+    // If forceRefresh is false, await.this.user() + resolve the token
+  };
```

Comment:
What is this meant to do? Function with no code inside

Reply:
Working on this in another branch